From 317f394e34bff53831bd8a7b18f58eb766108a61 Mon Sep 17 00:00:00 2001
From: Lennart Poettering <lennart@poettering.net>
Date: Tue, 9 Dec 2014 02:12:11 +0100
Subject: [PATCH] units: make sure container-getty@.service stops restarting
 when the pts device it is bound to is gone

We only want to restart the getty as long as the pts device is still
around. As soon as it is gone, the service should be removed to.

http://lists.freedesktop.org/archives/systemd-devel/2014-December/026048.html
(cherry picked from commit 68ac53e62fadb87f1b33ccd9bff9c3f7f699c937)
---
 units/container-getty@.service.m4.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/units/container-getty@.service.m4.in b/units/container-getty@.service.m4.in
index 4f7794b502..5120466929 100644
--- a/units/container-getty@.service.m4.in
+++ b/units/container-getty@.service.m4.in
@@ -14,6 +14,7 @@ After=rc-local.service
 )m4_dnl
 Before=getty.target
 IgnoreOnIsolate=yes
+ConditionPathExists=/dev/pts/%I
 
 [Service]
 ExecStart=-/sbin/agetty --noclear --keep-baud pts/%I 115200,38400,9600 $TERM
